spec/teacher_spec.rb

```diff
@@ -0,0 +1,19 @@
+require_relative '../classes/teacher'
+require_relative '../classes/classroom'
+
+describe Teacher do
+  teacher_one = Teacher.new(31, nil, 'francisco')
+  teacher_two = Teacher.new(17, nil, 'miguel')
+
+  context 'Testing the Teacher Class' do
+    it 'creates an instance of the teacher class' do
+      expect(teacher_one).to be_instance_of(Teacher)
+    end
+  end
+
+  context 'Test the can_use_service method' do
+    it 'return true by default in teachers' do
+      expect(teacher_two.can_use_services?).to eq(true)
+    end
```

Comment:
- [ OPTIONAL]  I recommend you to test  both positive and negative scenario case , for example to have  a test the check the scenario where a person is not allowed to use the services . It will help you to have all cases tested and avoid the future bugs from you application

Reply:
Thank you I'll keep it in mind in future testing, I really appreciate your suggestion. 😄